src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_backup_commands.py

```diff
@@ -357,7 +363,7 @@ def test_backup_rp(self, resource_group, vault_name, vm_name):
         })
 
         rp_names = self.cmd('backup recoverypoint list --backup-management-type AzureIaasVM --workload-type VM -g {rg} -v {vault} -c {vm} -i {vm} --query [].name', checks=[
-            self.check("length(@)", 2)
+            self.check("length(@)", 1)
```

Comment:
Why is it modified here?

Reply:
I removed a redundant test case from the test. So had to change it.